cvat/apps/dataset_manager/tests/test_annotation.py

```diff
@@ -165,4 +165,42 @@ def test_line_interpolation(self):
             ]
         }
 
-        self._check_interpolation(track)
\ No newline at end of file
+        self._check_interpolation(track)
+
+    def test_api_v1_unit_test_on_normalize_shape_function(self):
+        # 3 points
+        norm_shape = TrackManager.normalize_shape({
```

Comment:
@MashaSS , @yasakova-anastasia , why do we test "normalize_shape" function?

Reply:
It's the part of code related to dataset manager. It was decided to place it in `test_annotation.py` because test doesn't use rest api.

Reply:
@MashaSS , the function should be removed in the code. It isn't dead code. No need to cover it by tests. Just remove.

Reply:
Removed dead code function and test for this function.